packages/zk/build.sh

```diff
@@ -0,0 +1,46 @@
+# http(s) link to package home page.
+TERMUX_PKG_HOMEPAGE=https://github.com/zk-org/zk
+
+# One-line, short package description.
+TERMUX_PKG_DESCRIPTION=" A plain text note-taking assistant"
+
+# License.
+# Use SPDX identifier: https://spdx.org/licenses/
+TERMUX_PKG_LICENSE="GPL-3.0"
+
+# Who cares about package.
+# Specify yourself (Github nick, or name + email) if you wish to maintain the
+# package, fix its bugs, etc. Otherwise specify "@termux".
+# Please note that unofficial repositories are not allowed to reference @termux
+# as their maintainer.
+TERMUX_PKG_MAINTAINER="@kirasok"
+
+# Version.
+TERMUX_PKG_VERSION=0.14.0
+
+# URL to archive with source code.
+TERMUX_PKG_SRCURL=https://github.com/zk-org/zk/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+
+# SHA-256 checksum of the source code archive.
+TERMUX_PKG_SHA256=bd96f93d50e2e72ce05f36c3dab5b7942ae205756f26d4c68ba2a7ccc783abc8
+
+TERMUX_PKG_BUILD_DEPENDS="golang"
+
+TERMUX_PKG_BUILD_IN_SRC=true
+
+TERMUX_PKG_AUTO_UPDATE=true
+
+termux_step_pre_configure() {
+	termux_setup_golang
+
+	go mod init || :
+	go mod tidy
+}
+
+termux_step_make() {
```

Comment:
You don't need to override `termux_step_make`, the default one seems to work just fine.